src/main/src/test/java/org/geoserver/catalog/impl/CatalogImplTest.java

```diff
@@ -3564,6 +3564,7 @@ public void testExclusiveExactTermSearch() {
         cv.setName("waterways");
 
         addLayer();
+        catalog.add(cs);
         catalog.add(cv);
```

Comment:
If I'm not mistaken, this is ok but the prescribed way of doing so would be `addCoverage()` which already adds bod cs and cv.

Reply:
That will fail if you have already called addLayer(). That is why they didn't do it when they added these new testing methods. But they forgot this one line, so they created a layer without store. Jdbcconfig fails on that.

But this change shouldn't be here, that was an accident. It needs to be in a separate PR, because it has a different purpose, it's to make those new exact term tests work in jdbcconfig